lessons/frameworks.ipynb

```diff
@@ -0,0 +1,1261 @@
+    "  \n",
+    "* Bayesians seek to _update their beliefs_ about their model parameters, and hence quantify their uncertainty in terms of _what might have been had the observation been different_, and _what they knew before the data were taken_\n",
+    "\n",
+    "#### Q: Which approach is better matched to science?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Frequentism vs. Bayesianism\n",
+    "\n",
+    "\n",
+    "* \"You have to assume a prior\" cf. \"You get to assume a prior\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "* \"Your calculations are computationally expensive\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "* \"How do you account for your nuisance parameters?\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "* \"Your conclusions are not relevant\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Things to remember\n",
+    "\n",
+    "* The most important thing is to _know what you are doing_, and to _communicate that clearly to others_\n",
+    "\n",
+    "* Both approaches involve generative models, and their assumptions which must be recorded and tested\n",
+    "\n",
+    "* The Bayesian approach provides a logical framework for combining datasets and additional information, and provides answers in terms of the probability distribution for the model parameters\n",
+    "\n",
+    "* The Frequentist approach provides a way of studying the model independent of additional information, and provides answers in terms of the probability of getting the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Endnote\n",
+    "\n",
+    "* The astronomy literature contains a mixture of frequentist and Bayesian analysis, sometimes within the same paper\n",
+    "\n",
+    "* Frequentist estimators often make good _summary statistics_ with well understood sampling distributions: astronomical catalogs are full of them\n",
+    "\n",
+    "* In most of this course we follow the Bayesian approach: Bayes' Theorem gives you a framework for deriving the solution to _any_ inference problem you encounter.  Having said that, we'll keep our eyes open."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Appendix: Computing the Bayesian Posterior PDF\n",
```

Comment:
I'm not sure it's useful enough to have this stuff in the main notebook. I think we would normally put it in graphics/, although this is an unusual case because it does an analysis in addition to making figures.

Reply:
I needed it to make the figures of course - and then I thought, well, why not just leave it in as an appendix (rather than making a separate notebook in graphics). I think its OK - I won't go through it in class.